ouroboros-network/src/Ouroboros/Network/PeerSelection/RootPeersDNS/DNS/LedgerPeers.hs

```diff
@@ -1,35 +1,29 @@
-{-# LANGUAGE BangPatterns               #-}
-{-# LANGUAGE DerivingStrategies         #-}
-{-# LANGUAGE GADTs                      #-}
-{-# LANGUAGE GeneralizedNewtypeDeriving #-}
-{-# LANGUAGE OverloadedStrings          #-}
-{-# LANGUAGE RecordWildCards            #-}
-{-# LANGUAGE ScopedTypeVariables        #-}
-{-# LANGUAGE TypeApplications           #-}
-
-module Ouroboros.Network.PeerSelection.LedgerPeers
-  ( DomainAccessPoint (..)
-  , IP.IP (..)
-  , LedgerPeersConsensusInterface (..)
-  , RelayAccessPoint (..)
-  , PoolStake (..)
-  , AccPoolStake (..)
-  , TraceLedgerPeers (..)
+{-# LANGUAGE BangPatterns        #-}
+{-# LANGUAGE DerivingStrategies  #-}
+{-# LANGUAGE GADTs               #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE RecordWildCards     #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications    #-}
+
+module Ouroboros.Network.PeerSelection.RootPeersDNS.DNS.LedgerPeers
+  ( -- * Ledger Peers specific data types
+    UseLedgerAfter (..)
+  , IsLedgerPeer (..)
   , NumberOfPeers (..)
+  , LedgerPeersConsensusInterface (..)
   , LedgerPeersKind (..)
+    -- * Ledger Peers specific functions
   , accPoolStake
   , accBigPoolStake
-  , withLedgerPeers
-  , UseLedgerAfter (..)
-  , IsLedgerPeer (..)
-  , IsBigLedgerPeer (..)
-  , Socket.PortNumber
-    -- Re-exports for testing purposes
   , bigLedgerPeerQuota
+    -- * DNS based provider for ledger root peers
+  , ledgerPeersThread
+  , resolveLedgerPeers
```

Comment:
It seems we don't use `resolveLedgerPeers` outside of this module, except for tests.  I'd put it in an `* Internal` sections with a remark that it's only exposed for testing purposes.